### ISSUE TITLE
Displaying plots on map, with no property, choosing a colour and a size.

### ISSUE THREAD

**Issue by rvlander** [mapbox/mapboxgl-jupyter#13]
This appears not to be possible currently and it would be a huge gain.

**Comment by ryanbaumann** [comment]
@rvlander Sounds like you want to be able to display data with just point geometries, no property values, while selecting the default color and size?  

I think we should do this by exposing the [default Mapbox GL Style Spec property](https://www.mapbox.com/mapbox-gl-js/style-spec/#function-default) for a data-driven style.    This would allow you to specify a `default` circle radius, circle color, or any other style property in the case where there is no property value to parse for a data-driven style.

**Comment by rvlander** [comment]
@ryanbaumann, yes it is what would be need.

**Comment by ryanbaumann** [comment]
@rvlander excellent.  I'll add this as a feature for the next release.

**Comment by ryanbaumann** [comment]
Adding that we should support a default color profile if none is passed by the user.

**Comment by ryanbaumann** [comment]
This should be possible now using the `color_default` and `color_function_type` options:

```python
mapboxgl.viz.CircleViz(data, 
                                  color_default="red",
                                   color_function_type="match"
)
```

@rvlander can you confirm this works in the latest `mapboxgl` release on pip (0.1.4)?

**Comment by ryanbaumann** [comment]
Closed in #78
